raiden-network/light-client
# Feature/dapp adapt to arbitrum

**Thank you for submitting this pull request :)**

Fixes #3051 and #3053

**Short description**

I tackled both issues together, because only one of them would cause issues in the CI pipeline. Due to the underlying breaking changes of the SDK the dApp does not build properly without both changes together. As both issues are small enough, this should be fine.

Targets the `arbitrum` branch with the changes of the SDK.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Build the dApp successfully
2. Close a channel and verify that the button changes to `settle` but is disabled and no block countdown
3. Plan a withdrawal form the UDC and verify that is displays the date when it becomes ready, verify after the date passes it shows green
